### ISSUE TITLE
The data couldn’t be read because it isn’t in the correct format

### ISSUE THREAD

**Issue by Cachwir** [AppsFlyerSDK/appsflyer-cordova-plugin#235]
# Report

## Plugin Version

6.10.31 (reproduced also on 6.2.20)

## On what Platform are you having the issue?

iOS

## What did you do?

Called initSdk after opening the app through a deeplink

## What did you expect to happen?

The success function to be called and our deep link's related function to be called.

## What happened instead?

The failure function got called with the following payload : 
`jsonMessageStr = {"status":"failure","type":"onInstallConversionFailure","data":"The data couldn’t be read because it isn’t in the correct format."}`
Deep links on iOS only seem to be working when the app is already launched.

## Please provide any other relevant information.

We've been using this plugin for years now, and the deeplink was working as intended. And after about a year without publishing any update, we have decided to work on a new one. However, we've noticed that it doesn't work as intended anymore and can't seem to find the reason, no one here being skilled in Objective C.

**Comment by xmx77** [comment]
@ Cachwir
I have same issue , did you resolve it ?

**Comment by jetaggart** [comment]
@Cachwir @pazlavi any updates on this? we're running into the same issue

**Comment by pazlavi** [comment]
@Cachwir @xmx77 @jetaggart, 

If you receiving this error or 403 response in the logs, it might be because you have the Zero package. This package does not include attribution data, only data about clicks and impressions. For more information about our packages, contact the CEM team at [hello@appsflyer.com](mailto:hello@appsflyer.com) or visit [AppsFlyer pricing](https://www.appsflyer.com/pricing/).

For technical support, please submit a ticket to our support team at [support@appsflyer.com](mailto:support@appsflyer.com).

**Comment by jetaggart** [comment]
Yes this was a authorization issue with our account, we had to update our account with a new pricing tier, unfortunate the lib doesn't tell you that you're using a feature you don't have access too, very hard to figure out.

**Comment by raeesaa** [comment]
@jetaggart Thank you for sharing the solution. We are on the correct plan but still getting this error.

I will try to debug native code to figure out the root cause in our case.
